### ISSUE TITLE
Minimising PyQt based application in Python 

### ISSUE THREAD

**Issue by meetanubhav** [meetanubhav/email-scheduler#1]
**Minimizing a PyQt5 based python application to system dock or windows/system sub-process.**

Repo -:
https://github.com/meetanubhav/email-scheduler

Customization along with screenshots are accepted

**Comment by ghost** [comment]
Are you trying to mean minimizing by something like this?
```python
x =  "Hello!"
# to
x="Hello!"
```

**Comment by meetanubhav** [comment]
No. I want to minimise my application to windows docker. Like tou have
minimise tab in any application.
